ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig

```diff
@@ -5,11 +5,8 @@
 // https://github.com/dempseyatgithub/BuildSettingExtractor
 //
 
-ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
 CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES
-CLANG_ENABLE_MODULES = YES
-CLANG_WARN_DOCUMENTATION_COMMENTS = NO
-CODE_SIGN_IDENTITY = 
+CODE_SIGN_IDENTITY =
```

Comment:
why is this necessary ?

Reply:
these configurations cause the container compiling issues

Reply:
These are not required. Working for me on Debug config without these changes.

Reply:
are you able to build the debug container using new build system?

Reply:
Yes, I could build the Debug config mode without these changes.